### PR TITLE
Add info where to find security keys

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zwave_js.markdown [hunk 1]
```diff
@@ -771,6 +771,10 @@ S0 security imposes significant additional traffic on your mesh and is recommend
 
 S2 security does not impose additional network traffic and provides additional benefits, such as detecting packet corruption. By default, Z-Wave attempts S2 security during inclusion if supported, falling back to S0 security only when necessary.
 
+#### Where can I see the security keys in the Z-Wave JS add-on?
+
+After initial setup of the Z-Wave controller, you can view the security keys in the Z&#8209;Wave&nbsp;JS add-on. Go to <a class="my" href=" https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_zwave_js" target="_blank"><b>Settings</b> > <b>Add-ons</b> > <b>Z-Wave JS</b></a> and open the <b>Configuration</b> tab. You can now see nee the network security key as well as the S0 and S2 security keys.
```
Comment (frenck) [inline source/_integrations/zwave_js.markdown]:
```suggestion
After the initial setup of the Z-Wave controller, you can view the security keys in the Z-Wave JS add-on. Go to {% my supervisor_addon addon="core_zwave_js" title="**Settings > **Add-ons** > **Z-Wave JS**" %} and open the **Configuration** tab. You can now see the network security key and the S0 and S2 security keys.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by c0ffeeca7** [comment]
ah! that's how it is done. Thank you, @frenck !

File: source/_integrations/zwave_js.markdown [hunk 2]
```diff
@@ -771,6 +771,10 @@ S0 security imposes significant additional traffic on your mesh and is recommend
 
 S2 security does not impose additional network traffic and provides additional benefits, such as detecting packet corruption. By default, Z-Wave attempts S2 security during inclusion if supported, falling back to S0 security only when necessary.
 
+#### Where can I see the security keys in the Z-Wave JS add-on?
+
+After the initial setup of the Z-Wave controller, you can view the security keys in the Z-Wave JS add-on. Go to {% my supervisor_addon addon="core_zwave_js" title="**Settings > **Add-ons** > **Z-Wave JS**" %} and open the **Configuration** tab. You can now see the network security key and the S0 and S2 security keys.
```
Comment (AlCalzone) [inline source/_integrations/zwave_js.markdown]: Only 4 keys are relevant, the 3 S2 keys and the S0 key.
"Network key" is a legacy config setting, identical to the S0 key.

Comment (c0ffeeca7) [reply source/_integrations/zwave_js.markdown]: thank you, @AlCalzone :+1: . Updated section.

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**